### PR TITLE
Updated Docker to use php8.2-apache as the upstream image.

### PR DESCRIPTION
**Pull request by zampettim**
Updated to use php8.2-apache as the upstream image for Docker

### CODE REVIEW

**Review by Quentame — APPROVED** [review]

**Comment by stefanstidlffg** [comment]
i would use 
`FROM php:8-apache`
for better long term updates
works as well

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-apache
+FROM php:8.2-apache
```
Comment (stefanstidlffg) [inline Dockerfile:1]: FROM php:8-apache
will work too

**Comment by stefanstidlffg** [comment]
duplicate of #522

File: Dockerfile (hunk 1 above)
Comment (Quentame) [reply Dockerfile:1]:
```suggestion
FROM php:8-apache
```

**Review by Quentame — CHANGES_REQUESTED** [review]

**Comment by Quentame** [comment]
> duplicate of #522

Not really as this PR is focused in one subject, as a PR should be.
PR #522 is doing two different things, which is not a good practise.

**Comment by sstidl** [comment]
> > duplicate of #522
> 
> Not really as this PR is focused in one subject, as a PR should be. PR #522 is doing two different things, which is not a good practise.

point taken ;-)

**Review by sstidl — APPROVED** [review]

**Pull request merged**